### ISSUE TITLE
mail counter very hard to see

### ISSUE THREAD

**Issue by kilobyte** [gyunaev/birdtray#47]
Displaying the mail counter atop the Thunderbird icon makes the counter hard to notice — much less read the number it shows.  Even setting the color to something highly contrastive to colors of the icon (ie, black, white and blue) doesn't seem to help:

![birdtray-icon](https://user-images.githubusercontent.com/48801/49704160-5e306280-fc07-11e8-88e8-3bb072fa2b82.png)

Firetray avoid this by showing the numer atop a completely blank icon (with nothing but a frame and a gradient).

There's an option in Birdtray to use a different icon when unread, but 1. it's off by default, 2. no icons are provided, and 3. it requires a specific size — for example, trying to using Firetray's icon results in:
![bird-fire](https://user-images.githubusercontent.com/48801/49704222-22e26380-fc08-11e8-8cd4-33ec632ae418.png)

**Comment by jtrulson** [comment]
I had the same issue... I couldn't read the number very well overlayed onto the thunderbird icon.

So I just created a 32x32 .png with a white background and a black, single pixel border (so any font color other than white should be clearly visible).  It works well for me (I have attached it).

It would be nice if this, and maybe other icons were included in the images qrc file with the thunderbird icon for easy selection, But this works well enough for me.

![square](https://user-images.githubusercontent.com/10564998/49968451-1b121000-fee3-11e8-9ce2-454008ff774d.png)
